ios/analytics.md

```diff
@@ -58,7 +58,7 @@ The Amplify CLI helps setup and configure Pinpoint within your application and c
 
 #### Update your IAM Policy:
 
-Amazon Pinpoint service requires an IAM policy in order to use the `submitEvents` API:
+Amazon Pinpoint service requires an IAM policy in order to use the `submitEvents` API. If you are using long-term AWS credentials of an `Amazon IAM` user, attach the following policies to the role of the `IAM` user. If you are using temporary AWS credentials vended by `Amazon Cognito Identity Pools`, attach the following policies to the Unauthenticated and/or Authenticated `IAM` roles of your `Cognito Identity Pool`.
```

Comment:
Is this a duplicate of the above text? If so just use the same above

Reply:
Yes it is. Updated the same.